### DISCUSSION TITLE
Can I delete an item in TTKDesigner?

### DISCUSSION THREAD

**Discussion by SZRabinowitz** [ceccopierangiolieugenio/pyTermTk#297] [Q&A] [answered]
Hi, great project! This is the best tui library for python I've seen!

In ttkdesigner, is it possible to remove a widget (button, windows, etc) that I added?

**Comment by SZRabinowitz** [comment, marked as answer]
Never mind. After some struggling I got it to work with 'del' button

**Comment by ceccopierangiolieugenio** [comment]
oooh, Thanks :-D

yes, so far I mapped only the 'Del' button,
TBH since the library supports the right mouse button submenu, I should use it more extensively to perform different actions on the widgets.
anyway, I am always looking forward for suggestion and improvements, unfortunately I am not spending much time on the designer because any time I touch it I feel I should refactor it completely.

Thanks!!!

**Reply by SZRabinowitz** [reply]
No problem. In the end, I might not use the designer, because I like creating objects that I have direct access to. 


Unrelated: is it possible to color the borders of a window or other widgets?

**Reply by ceccopierangiolieugenio** [reply]
yes you can;
The reason I still haven't advertised it is because it is an alpha feature and I stil have to properly finalize the api.
Anyway in the docs there is the default style definition of any widget that support it.

```python
root = ttk.TTk(title="pyTermTk Styles Demo")

win = ttk.TTkWindow(parent=root, size=(50,10), title=" Test Window ")
win.mergeStyle(
        {'default':{
                    'color':ttk.TTkColor.FG_RED+ttk.TTkColor.BG_GREEN,
                    'fillColor':ttk.TTkColor.BG_BLACK,
                    'borderColor':ttk.TTkColor.MAGENTA},
         'focus':{
                    'color':ttk.TTkColor.BG_RED+ttk.TTkColor.FG_GREEN,
                    'fillColor':ttk.TTkColor.BG_BLUE,
                    'borderColor':ttk.TTkColor.CYAN}})

root.mainloop()
```
**NOTE:** I just noticed that **fillColor** is not implemented in the window object, I just added it and it will be available in the next patch release.

https://github.com/user-attachments/assets/389e4708-83de-43ea-a13b-9431bfcd93ec

**Reply by SZRabinowitz** [reply]
Wonderful, thank you so much!


This library has a lot of potential! I have been through a few TUI libraries in Python, and this is the best. PyTermGui just isn't smooth. And textual does not offer window management. PyTermTk is amazing for complex applications with the ability to spawn windows and tabs, etc on demand. 


I am going to try building a TUI client for the GroupMe Messaging platform and I will let you know if I have any issues or questions.

**Comment by ceccopierangiolieugenio** [comment]
Wow thanks very much, Sometimes I feel shadowed by my competitors 😄 
and I progress with my library out of stubbornness and simple curiosity.

Of course, if you need any help or new features or someone capable to dig into the intricacies of my library, just let me know, 
I am really looking forward to see your app.
